en/user_guide/content_display.rst

```diff
@@ -0,0 +1,64 @@
+Contents display
+================
+
+Once contributed, there is only one way to display a content: using content blocks in nodes. Open
+Orchestra comes with 3 block types able to render a content, but a developper can create his own block
+to match its need. The 3 standard Open Orchestra content blocks are:
+
+* Content list block
+* Content block
+* Configurable content block
+
+
+Content list block
+------------------
+This block renders in Front Office a list of contents matching criterias given by the setting of the
+block. A typical use may be to display a list of short previews of several articles, or a listing of
+customers.
+
+Contribution
+````````````
+Besides the common settings for all blocks, the content list block requires two specific types of
+settings:
+
+* Criterias: These are a combination of two basic criterias. The contributor must choose a content
+  type and/or keyword(s) to trigger the contents.
+
+* Rendering: The contributor has two ways to render this block. He can use the standard template of
+  the content list block or create one in the Back Office to be used in the context of that node. To
+  use the standard template, put the template usage on Off. To use a unique node template, put the
+  template usage on On, then use the rich text below to create the template. You can use the twig syntax.
+  This template will be used only for that block in that node. Other content list blocks in that node
+  or in others are not been concerned by that template.
+
+
+Content block
+-------------
+The content block renders the content whose id is given in the address of the page. A typical use is
+to create a generic node rendering specific content detailed information. By varying the content id
+in the url, the same node can be used to display various articles or customer details. For instance
+a node with the pattern /customer/{contentId} would be accessed in Front Office via url /customer/
+paul-smith and then the content block would display information about Paul Smith. It could also be
+accessed via url /customer/john-anderson, and then displaying information about John Anderson.
+
+Contribution
+````````````
+On this block, like on the content list block, the contibutor can use the standard template of the
+block or create a specific one for that block/node using the rich text.
+
+**To get this block working, it's important to set the node pattern in a specific way. As the content
+id comes from the url, the node pattern must include the string {contentId} or the content id will
+not be available**
+
+
+Configurable content block
+--------------------------
+The configurable content block renders a single content, chosen by the contributor. A typical use is
+to create a focus on a specific content somewhere in the page, that could not be the principal
+information of the page, such has a mini-block on the last product developped by the company, or the
+employee of the month.
```

Comment:
Hum it could be yes. But for now it's hard to trigger a specific content with content list block. Plus as the name says, it should be used to display a list, not a single content. Maybe with something visual to separate each content that would make no sense in the context of a single content.